### ISSUE TITLE
DigiCert One KeyLocker documentation

### ISSUE THREAD

**Issue by ennerf** [hydraulic-software/conveyor#77]
I had to renew a DigiCert certificate and migrated to the new KeyLocker integration. IMO their documentation made it more cumbersome than it needed to be, so here is a walkthrough in case other users are struggling with the same thing:

**1) Create a DigiCert ONE account**
The first time when selecting KeyLocker you'll get an email to create a new account for [DigiCert ONE](https://one.digicert.com/). Create the account when you receive it as the link expires after 3 days and there is no way to get another one without contacting support.

**2) Generate auth keys**
First, you need to switch to the Admin Profile on the top right

![image](https://github.com/hydraulic-software/conveyor/assets/5491587/06d66297-e73e-4848-9225-90f4329fab29)

In the Access menu generate an auth certificate (values for `digi-cert-one.auth-certificate` and `digi-cert-one.password`), as well as an API token (value for `digi-cert-one.api-key`)
![digicert-auth-tokens](https://github.com/hydraulic-software/conveyor/assets/5491587/b702436d-68da-4bf2-9b0d-64b530bcad56)

**3) Switch to KeyLocker**

Another menu switch that is hidden on the grid in the top right menu
![digicert-keylocker](https://github.com/hydraulic-software/conveyor/assets/5491587/276fe66c-4f40-47e2-a0ce-3e93c7cf3056)

**4) Find the certificate alias**

You can skip the Get Started tutorial and go straight to certificates. If the order status shows `pending`, you need to run `synchronize orders` next to the order ID.

![digicert-cert-alias](https://github.com/hydraulic-software/conveyor/assets/5491587/07a8a952-521c-446b-99bb-523977d15d80)

Copy the keypair alias `key_<number>`, replace `key_` with `cert_`, and set it as the value for `digi-cert-one.signing-key-alias`. Conveyor requires the certificate alias, but that is only visible from the CLI tool. 

**Full configuration**
```hocon
app.windows {
  signing-key {
    digi-cert-one {
      auth-certificate = <path to the auth certificate p12 file>
      password = <password for the p12 file>
      api-key = <api token>
    }
  }
  signing-key-alias = cert_<number>
}
```

**Comment by ennerf** [comment]
Integrating the auth certificate .p12 file into Github CI secrets can be done by going through base64.

**1) Store binary content as a base64 secret**
```bash
base64 <certificate_name>.p12 | pbcopy
```

**2) Decode the binary contents to a temporary file**

Using [timheuer/base64-to-file](https://github.com/timheuer/base64-to-file)

```yaml
 - name: Create DigiCert auth cert
   id: auth_cert
   uses: timheuer/base64-to-file@v1.2
   with:
     fileName: 'DigiCert_auth.p12'
     encodedString: ${{ secrets.DIGICERT_AUTH_CERT_BASE64 }}
```

**3) Pass the temporary file path as an environment variable**

```yaml
 - name: Conveyor Package
   uses: hydraulic-software/conveyor/actions/build@v11.0
   env:
     DIGICERT_AUTH_CERT: ${{steps.auth_cert.outputs.filePath}}
```

**Comment by mikehearn** [comment]
Thanks again Florian, we released this as part of 11.1
